.github/workflows/ci_workflows.yml

```diff
@@ -143,39 +146,41 @@ jobs:
         sudo apt-get update
         sudo apt-get install language-pack-de tzdata
     - name: Install Python dependencies
-      run: python -m pip install --upgrade tox codecov
+      run: python -m pip install --upgrade tox
     - name: Run tests
       run: tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
 
-  parallel_and_32bit:
-    name: 32-bit and parallel
+  parallel-double:
+    name: Parallel and double
     runs-on: ubuntu-latest
     needs: initial_checks
-    container:
-      image: quay.io/pypa/manylinux2014_i686
```

Comment:
Here try adding ``:2022-04-03-da6ecb3`` at the end

Reply:
OK let's see...